File: javascript/1.js
// project euler #1 in javascript by antekone
// http://anadoxin.org/blog
//
// deps: node.js
//
// usage:
// $ node 1.js

var sum = 0;

for(var i = 0; i < 1000; ++i) {
    if(i % 3 == 0 || i % 5 == 0) {
        sum += i;
    }
}

console.log(sum);

File: pascal/1.pas
{
    project euler #1 in pascal by antekone
    http://anadoxin.org/blog

    usage:

    $ fpc 1.pas
    $ ./1
}

program euler1;
var
    sum, i: UInt32;

begin
    sum := 0;

    for i := 0 to 999 do
        if (i mod 3 = 0) or (i mod 5 = 0) then
            sum := sum + i;

    writeln(sum)
end.
